templates/simulation.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'base.html' %}
+{% block content %}
+	<div class="row">
+		<div class="col-md-5">
+				<h2 class="page-header">{% block title %}Run Simulation{% endblock %}</h2>
+				<form method="post" class="form-horizontal" action="/jobs">
+	            	<div class="form-group">
+	                	<div class="col-sm-2"></div>
+	            	</div>
+	            	<label for="nc">Number of Cores:</label>
+	            	<input type="text" class="form-control" name="cores" id = "nc" value="{{request.form.cores}}">
```

Comment:
I think the names here should be the same as in /data/default_simulator_parameters.plist.